services/ui-src/src/components/fields/ChoiceListField.test.tsx

```diff
@@ -224,10 +227,13 @@ describe("<ChoiceListField />", () => {
 
     test("CheckboxField should render nested child fields for choices with children", async () => {
       // Render Initial State and choices
+      mockGetValues(undefined);
       render(CheckboxComponentWithNestedChildren);
       expect(screen.getByText("Choice1")).toBeVisible();
       expect(screen.getByText("Choice2")).toBeVisible();
       expect(screen.getByText("Choice3")).toBeVisible();
+      expect(screen.queryByText("Choice4")).not.toBeInTheDocument();
+      expect(screen.queryByText("Choice5")).not.toBeInTheDocument();
```

Comment:
Adding these tests so we don't have a false positive when the elements should appear after click.